docs/sources/operations/zone-ingesters.md

```diff
@@ -0,0 +1,96 @@
+5. enable zone awareness on the read path by setting `distributor.zone-awareness-enabled` to true for queriers
+6. configure distributors and rulers to exclude ingesters in the `zone-default` so those ingesters no longer receive write traffic via `distributor.excluded-zones`
+7. use the shutdown endpoint to flush data from the default ingesters, then scale down and remove the associated statefulset
+8. clean up
+
+
+### jsonnet
+The following are steps to live migrate (no downtime) an existing Loki deployment from a single ingester statefulset to 3 zone aware ingester statefulsets. 
+
+These instructions assume you are using the zone aware ingester jsonnet deployment code from this repo, see [here](https://github.com/grafana/loki/blob/main/production/ksonnet/loki/multi-zone.libsonnet). If you are not using jsonnet please see relevant annotations in some steps that describe how to perform that step manually.
+
+1. Configure the zone for the existing “ingester” StatefulSet as zone-default by setting multi_zone_default_ingester_zone: true, this allows us to later filter out that zone from the write path.
+2. Configure ingester-pdb with maxUnavailable=0 and deploy 3x zone-aware StatefulSets with 0 replicas by setting
+```
+ _config+:: {
+    multi_zone_ingester_enabled: true,
+    multi_zone_ingester_migration_enabled: true,
+    multi_zone_ingester_replicas: 0,
+    // These last two lines are necessary now that we enable zone aware ingester by default
+    // so that newly created cells will not be migrated later on. If you miss them you will
+    // break writes in the cell.
+    multi_zone_ingester_replication_write_path_enabled: false,
+    multi_zone_ingester_replication_read_path_enabled: false,
+  },
+```
+
+If you're not using jsonnet the new ingester statefulsets should have a label with `rollout-group: ingester`, annotation `rollout-max-unavailable: x` (put a placeholder value in, later you should set the value of this to be some portion of the statefulsets total replicas, for example in jsonnet we template this so that each statefulset runs 1/3 of the total replicas and the max unavailable is 1/3 of each statefulsets replicas), and set the update strategy to `OnDelete`.
+2.  Diff ingester and ingester-zone-a StatefulSets and make sure all config matches 
+```
+kubectl get statefulset -n loki-dev-008 ingester -o yaml > ingester.yaml
+kubectl get statefulset -n loki-dev-008 ingester-zone-a -o yaml > ingester-zone-a.yaml
+diff ingester.yaml ingester-zone-a.yaml
+```
+expected diffs are things like: creation time and revision #, the zone, fields used by rollout operator, # of replicas, anything related to kustomize/flux, and PVC for the wal since the containers don't exist yet
+3. Temporarily double max series limits for users that are using more than 50% of their current limit, the queries are as follows (add label selectors as appropriate):
```

Comment:
Step 3 is not on it's own line in the preview, I think because step 2 is missing a period at the end?